views/partials/header.ejs

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<!--- Head Section --->
+<head>
+    <title>Botanical Lens - <%= title %></title>
+    <link rel="stylesheet" href="./css/bootstrap.min.css">
+    <link rel='stylesheet' href='/stylesheets/partials/header_style.css'/>
+    <link rel='stylesheet' href='/stylesheets/style.css'/>
+</head>
+
+<!--- Body Section --->
+<body>
+
+<!--- Header Section --->
+<div class="popover-header">
+    <header class="d-flex flex-wrap align-items-center justify-content-center justify-content-md-between bg-white">
+
+        <!-- Company Branding -->
+        <a onclick="location.href='/'"
+           class="d-flex align-items-center col-md-3 text-dark text-decoration-none"
+           style="font-size: xx-large">
+            <img src="../images/logo/botanical-lens-logo.png"
+                 alt="Botanical Lens" class="" width="80" height="80" style="margin-left: 7%; margin-top: 2%"/>
+            <span class="gradient-text logo-title-gradient"> Botanical Lens </span>
+        </a>
+
+        <!--- Search Box --->
+        <form class="d-flex ms-1 border rounded-2 w-50">
+            <input class="form-control" id="searchbox" type="search" placeholder="Search for a Plant"
+                   aria-label="Search" style="border-color: #3C1414">
+        </form>
+
+        <!-- Navigation Options -->
+        <ul class=" nav col-12 col-md-auto mb-2 justify-content-center mb-md-0">
+            <li><a href="#" class="nav-link px-2"
+                   style="font-size: large; color: #3C1414">Add Plant</a>
+            </li>
+            <li><a onclick="location.href='/'" class="nav-link px-2"
```

Comment:
lets try to be consistent. As a good practice, lets use href attribute.

Reply:
Not my code, Just added the colours.